frontend/src/components/sign_up/sign_up.css

```diff
@@ -1,130 +1,99 @@
-}
 
-html,
-body {
-  height: 100%;
-}
+.firstName.col-md.form-control-lg {
+  border: .05rem;
+  border-style: solid;
+  border-color: #cfcfcf;
 
-body {
-  display: -ms-flexbox;
-  display: -webkit-box;
-  display: flex;
-  -ms-flex-align: center;
-  -ms-flex-pack: center;
-  -webkit-box-align: center;
-  align-items: center;
-  -webkit-box-pack: center;
-  justify-content: center;
-  padding-top: 40px;
-  padding-bottom: 40px;
-  background-color: #f5f5f5;
 }
 
-.form-signUp {
-  width: 100%;
-  max-width: 400px;
-  padding: 15px;
-  margin: 0 auto;
+#inputLastName {
+  border: .05rem;
+  border-style: solid;
+  border-color: #cfcfcf;
 }
 
-.form-label-group>input,
-.form-label-group>label {
-  padding: var(--input-padding-y) var(--input-padding-x);
+.signUpButton.btn.btn-lg.btn-block {
+  background-color:#06A6D3;
+  color: white;
 }
 
-.form-label-group>label {
-  position: absolute;
-  top: 0;
-  left: 0;
-  display: block;
+.fullWidth.form-control-lg {
+  border: .05rem;
+  border-style: solid;
+  border-color: #cfcfcf;
   width: 100%;
-  margin-bottom: 0;
-  /* Override default `<label>` margin */
-  line-height: 1.5;
-  color: #495057;
-  border: 1px solid transparent;
-  border-radius: .25rem;
-  transition: all .1s ease-in-out;
 }
 
-.form-label-group input::-webkit-input-placeholder {
-  color: transparent;
-}
-
-.form-label-group input:-ms-input-placeholder {
-  color: transparent;
+.partWidth.form-control-lg {
+  width: 100%;
 }
 
-.form-label-group input::-ms-input-placeholder {
-  color: transparent;
+.fas.fa-eye.fa-lg, .fas.fa-eye-slash.fa-lg {
+  color:#cfcfcf;
+  position: absolute;
+  margin-left: 90%;
+  margin-top: 4%;
 }
 
-.form-label-group input::-moz-placeholder {
-  color: transparent;
+.password.form-group.row {
+  position: relative;
+  margin-right: 0;
 }
 
-.form-label-group input::placeholder {
-  color: transparent;
+#inputFirstName:focus, #inputLastName:focus, #inputSignUpPassword:focus, #inputSignUpEmail:focus{
+  outline: none;
 }
 
-.form-label-group input:not(:placeholder-shown) {
-  padding-top: calc(var(--input-padding-y) + var(--input-padding-y) * (2 / 3));
-  padding-bottom: calc(var(--input-padding-y) / 3);
+form > .form-group.row {
+  position: relative;
+  caret-color: #cfcfcf;
+  margin-right: 0;
+  
 }
 
-.form-label-group input:not(:placeholder-shown)~label {
-  padding-top: calc(var(--input-padding-y) / 3);
-  padding-bottom: calc(var(--input-padding-y) / 3);
-  font-size: 12px;
-  color: #4EC8ED;
+form > .form-group.row > label {
+  position: absolute;
+  padding-left: 1rem;
+  padding-top: 1rem;
+  font-size: small;
+  color: #cfcfcf;
+  margin-right: 0;
 }
 
-button {
-  background-color: #4EC8ED;
-  color: white;
+#inputSignUpEmail:focus + label, #inputSignUpPassword:focus + label, #inputFirstName:focus + label, #inputLastName:focus + label {
+  bottom: 45%;
+  font-size: x-small;
+  color: #06A6D3;
 }
 
-a {
-  color: #495057;
-  text-decoration: underline;
+.form-group.row > input {
```

Comment:
How about just `.form-signup input`?

Reply:
I tried . . . there's a css conflict somewhere making the formatting disappear. . .